README.md

```diff
@@ -82,13 +82,17 @@ k8gb is very well tested with the following environment options
 
 | Type                             | Implementation                                                          |
 |----------------------------------|-------------------------------------------------------------------------|
-| Kubernetes Version               | >= 1.14 (with [install workaround][install-workaround] >= 1.15 (Stable) |
-| Environment                      | On-prem(vSphere), AWS(EKS)                                              |
-| Ingress Controller               | NGINX                                                                   |
-| EdgeDNS                          | Infoblox, Route53                                                       |
+| Kubernetes Version               | >= 1.15                                                                 |
+| Environment                      | On-prem(vSphere), AWS(EKS) [*](#clarify)                                |
```

Comment:
```suggestion
| Environment                      | Self-managed, AWS(EKS) [*](#clarify)                                |
```

@ytsarev wondering if we should just mention "Self-managed" Kubernetes or something instead of `On-prem(vSphere),`? The fact that it is Kubernetes on top of vSphere is not really relevant, just that it is not a _managed_ Kubernetes service like EKS/AKS/GKE etc. but rather self-managed.

Would rather not give the impression that there is anything reliant on vSphere in any way.

Reply:
@donovanmuller great point, thanks.